x-pack/plugins/logstash/server/lib/check_license/check_license.test.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { licensingMock } from '../../../../licensing/server/mocks';
+import { checkLicense } from './check_license';
+
+describe('check_license', function() {
+  describe('returns "valid": false & message', () => {
+    it('license information is not available', () => {
+      const license = licensingMock.createLicenseMock();
+      license.isAvailable = false;
+
+      const { valid, message } = checkLicense(license);
+
+      expect(valid).toBe(false);
+      expect(message).toStrictEqual(expect.any(String));
```

Comment:
We don't want to assert against the various messages possible values in these tests?

Reply:
I don't think any changes in i18n dictionaries should fail tests. WDYT?

Reply:
Right, forgot these were user-facing and therefor i18nized. nm then.